tests/ztest/repeat/testcase.yaml

```diff
@@ -0,0 +1,26 @@
+tests:
+  testing.ztest.repeat_suite_regex:
+    platform_allow: qemu_x86
+    harness: console
```

Comment:
ztest_suite_repeat and ztest_test_repeat are configuration options of ztest harness, not console.
How is this going to work when used in other tests? does each ztest need to start parsing console and output?